datastore/api/README.md

```diff
@@ -0,0 +1,4 @@
+# Cloud Datastore API Samples
+
+<!-- auto-doc-link -->
+<!-- end-auto-doc-link -->
```

Comment:
It's for a [tool](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/scripts/auto_link_to_docs.py) that takes a list of links to this project in https://cloud.google.com and sticks them into the readme. The part of the tool that generates the list of links is internal. :(